LiskHQ/lisk-desktop
# Enable auto-updater for Linux - Closes #2740

### What was the problem?
This PR resolves #2740



### How was it solved?

- [x] Delete the condition that disables auto-updater on Linux
- [x] Update Jenkinsfile to archive `latest-linux.yml`

TODO after this PR is merged: Update our Release document process to mention that also `latest-linux.yml` file must be copied over from Jenkins to the Github release when doing a release.


### How was it tested?

On Linux:
- Check out this branch
- Decrease `version` in `app/package.json`. E.g. to `1.24.0-beta.0`
- Build the app. E.g. with `npm run pack`
- Open the app to see the new version flash message
<img width="808" alt="Screenshot 2020-02-17 at 15 06 16" src="https://user-images.githubusercontent.com/1254342/74660807-2459d500-5197-11ea-90a7-618d25bf0419.png">
<img width="807" alt="Screenshot 2020-02-17 at 15 06 37" src="https://user-images.githubusercontent.com/1254342/74660817-2885f280-5197-11ea-99e1-29b416d2a084.png">


From any OS:
- Check that Jenkins build artefacts contain `latest-linux.yml`, e.g.: https://jenkins.lisk.io/blue/organizations/jenkins/lisk-desktop/detail/2740-autoupdater-for-linux/2/artifacts
